README.md

```diff
@@ -113,10 +113,11 @@ module.exports = {
 | :-----------------------------------: | :-------------------------: | :----------------: | :--------------------------------------------------------------------- |
 |           **[`url`](#url)**           |    `{Boolean\|Function}`    |       `true`       | Enables/Disables `url`/`image-set` functions handling                  |
 |        **[`import`](#import)**        |    `{Boolean\|Function}`    |       `true`       | Enables/Disables `@import` at-rules handling                           |
-|       **[`modules`](#modules)**       | `{Boolean\|String\|Object}` |      `false`       | Enables/Disables CSS Modules and their configuration                   |
+|       **[`modules`](#modules)**       | `{Boolean\|String\|Object}` |   `{auto: true}`   | Enables/Disables CSS Modules and their configuration                   |
+|          **[`icss`](#icss)**          |         `{Boolean}`         |    `!!modules`     | Enables/Disables Interoperable CSS                                     |
```

Comment:
I don't know the best best way to indicate that the default value for `icss` depends on the `modules` option.

Reply:
Rewrite it `false by default, otherwise depend on `modules``